data_scraping/tensorflow_data.py

```diff
@@ -323,12 +356,21 @@ def getDates(start, end):
     return dates
 
 
+# if __name__ == "__main__":
+#     print("start")
+#     startDate = sys.argv[1]
+#     endDate = sys.argv[2]
+#     dates = getDates(startDate, endDate)
+#     dataFolderPath = sys.argv[3]
+#     for i in range(len(dates) - 1):
+#         print(dates[i], dates[i + 1])
+#         main(dates[i], dates[i + 1], dataFolderPath)
+
```

Comment:
Shouldn't this be uncommented or modified somehow because the get dates functionality won't run.
If not required get rid of the useless comments.

Reply:
Yes, it has been uncommented and other necessary changes are made also the months are broken into smaller chunks for the API limit problem.